### ISSUE TITLE
Serializer tests for status and publish statuses

### ISSUE THREAD

**Issue by data-sync-user** [mozilla/experimenter#7983]
Our current implementation of the lifecycle states and publish statuses are not fully covered by our unit tests. Let's get these transitions covered before we work on DIRTY editability.

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/EXP-2952)

**Comment by data-sync-user** [comment]
➤ Elise Richards commented:

There’s some conflicts between what is defined in our sequence diagrams and state diagrams and what is actually implemented. This review has been on hold for a few days waiting for Jared’s return so we don’t accidentally miss something / break something due to missing context. Should be unblocked now 👍
